ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/submission/NonTransientContractsITSpec.scala

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.submission
+
+import com.daml.ledger.api.benchtool.config.WorkflowConfig
+import com.daml.ledger.api.benchtool.services.LedgerApiServices
+import com.daml.ledger.api.benchtool.BenchtoolSandboxFixture
+import com.daml.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
+import com.daml.ledger.client.binding
+import com.daml.scalautil.Statement.discard
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.{AppendedClues, Checkpoints, EitherValues, OptionValues}
+
+import scala.concurrent.Future
+
+class NonTransientContractsITSpec
```

Comment:
Is the focus of this test non-transient or transient contracts (non-transient being the usual kind).

Reply:
Transient contracts were the default and the only choice before. So here the focus is on the non-transient contracts